### PR TITLE
added GET request and .env file for hiding API key

### PR DESCRIPTION
**Pull request by mswarts**

### CODE REVIEW

File: APIcall.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const axios = require('axios');
+
+var app = express();
+
+app.get("/results", function(req,res) {
+    axios("https://api.openweathermap.org/data/2.5/weather?q=Seattle&appid=34e47a9c09f7eed541ec45ab96dda7e8", function(error, response, body) {
```
Comment (stuttskl) [inline APIcall.js]: Let's remove this for now :) While testing, I can use my own API key

Comment (stuttskl) [reply APIcall.js]: Sorry, by "this" I mean the API key haha

**Review by stuttskl — CHANGES_REQUESTED** [review]

File: APIcall.js (hunk 1 above)
Comment (tdraxler) [reply APIcall.js]: Seconded, with bots that scan github for API keys going around and all. haha

**Review by tdraxler — CHANGES_REQUESTED** [review]
Good start! I like that you put the API call in a separate file. Also need to see if we get weather results (is results defined?)
Tomorrow we can go over:
-Turning APIcall.js into a route file and importing its contents into the main server app file.
-directory structure
-Testing the route to see what we get.

File: routes/APIcall.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          res.sendStatus(400);
+      })
+      
+    //if(!error && response.statusCode == 200){
+            //var parsedData = JSON.parse(body);
+           // res.send(results["temp"]);
+      //  }
+   // });
+    });
+//});
```
Comment (tdraxler) [inline routes/APIcall.js]: Looks good! Just need to fix the indentation and remove the comments

**Review by tdraxler — APPROVED** [review]

**Review by stuttskl — APPROVED** [review]
Looks great!

**Pull request merged**